src/handlers/miner_dkg_handler.erl

```diff
@@ -312,8 +312,7 @@ serialize(State) ->
            sent_conf => SentConf,
            height => Height,
            delay => Delay},
-    M = maps:map(fun(_K, Term) -> t2b(Term) end, M0),
-    maps:merge(PreSer, M).
+    maps:merge(PreSer, M0).
```

Comment:
PreSer is an empty map. Do we need the `maps:merge`?

Reply:
No, good catch.